paywall/src/data-iframe/index.2.0.ts

```diff
@@ -21,10 +56,18 @@ const web3Service = new Web3Service({
 // method in the model is called.
 let blockchainReader = new BlockchainDataStorable()
 
-const initializeReader = (lockAddresses: string[], accountAddress: string) => {
+interface ReaderArgs {
+  lockAddresses: string[]
+  accountAddress: string
+}
+const initializeReader = ({ lockAddresses, accountAddress }: ReaderArgs) => {
```

Comment:
Had an issue with calling `initializeReader` from the other end with multiple arguments, so now we pass an object to it.